docs/pages/_app.tsx

```diff
@@ -1,6 +1,8 @@
 import { GlowProvider } from "@glow-app/glow-react";
 import "@glow-app/glow-react/dist/styles.css";
 
+import { NetworkProvider } from "../components/NetworkContext";
```

Comment:
`_app.tsx` has three paths right now. I think it should only have two.

Reply:
Yep! I’m opening a separate PR for that soon, I didn’t want to add more to this PR.